Strings/Z.cpp

```diff
@@ -0,0 +1,34 @@
+#include<bits/stdc++.h>
+
+using namespace std;
+
+vector<int> get_z(string s) {
+    int n = s.size();
+    vector<int> z(n);
+    int l = 0, r = 0;
+    for (int i = 1; i < n; i++)
```

Comment:
nit: Prefer curly brackets around multiline blocks for readability.

Reply:
Nah